weblate/settings_example.py

```diff
@@ -839,6 +839,11 @@
     "weblate.wladmin.tasks.backup_service": {"queue": "backup"},
 }
 
+# Enable to skip backups
+
+BACKUP_SKIPPED = False
+BACKUP_COMPRESSED = False
```

Comment:
* I think it should include what kind of backup is this about, for example `DATABASE_BACKUP`.
* Maybe having just one option would make it easier (`DATABASE_BACKUP` with options (`none`, `plain`, `custom`).
* Please define new setting in [related AppConf](https://github.com/WeblateOrg/weblate/blob/664459eed0d7404de7e83e871b98e4305fdeb10d/weblate/utils/models.py#L32-L66) so that the setting is always present and the code doesn't have to deal with situation it is not present.

Reply:
> Maybe having just one option would make it easier (DATABASE_BACKUP with options (none, plain, custom).

I hesitated but ok, but I'd prefer the values `none`, `plain`, `compressed` then I guess.

> Please define new setting in related AppConf so that the setting is always present and the code doesn't have to deal with situation it is not present.

Thanks, I was wondering about something like this.

Reply:
My idea was to stay consistent with pg_dump options and to be able to use the option directly there.